source/iot/Mqtt5Client.cpp

```diff
@@ -313,6 +313,12 @@ namespace Aws
             return *this;
         }
 
+        Mqtt5ClientBuilder &Mqtt5ClientBuilder::WithBootstrap(Crt::Io::ClientBootstrap *bootStrap) noexcept
+        {
+            m_bootstrap = bootStrap;
```

Comment:
As the builder does not do any extra operations on bootstrap, we would directly pass along the bootstrap to the options. The member variable `m_bootstrap` would not be necessary either. 
```
m_options->WithBootstrap(bootStrap);
```

Reply:
oh, that is true, I just saw the member and didn't think about it. Actually the member wasn't even used anywhere so it was not even using the bootstrap. I fixed it now, thanks!